search/controllers/api/__init__.py

```diff
@@ -45,18 +53,23 @@ def search(params: MultiDict) -> Tuple[Dict[str, Any], int, Dict[str, Any]]:
     """
     q = APIQuery()
 
-    # parse advanced classic-style queries
+    # parse NG queries utilizing the Classic API syntax.
+    # This implementation parses the `query` parameter as if it were
+    # using the Classic endpoint's `search_query` parameter. It is meant
+    # as a migration pathway so that the URL and query structure aren't
+    # both changed at the same time by end users.
+    # TODO: Complete this implementation.
```

Comment:
What is left to complete?

Reply:
The behavior when specifying an id_list is not yet implemented.

Reply:
This is done.